optuna_dashboard/_app.py

```diff
@@ -347,33 +349,59 @@ def send_static(filename: str) -> BottleViewReturn:
         return static_file(filename, root=STATIC_DIR)
 
     register_rdb_migration_route(app, storage)
-    register_artifact_route(app, storage, artifact_backend)
+    register_artifact_route(app, storage, artifact_store)
     return app
 
 
 def run_server(
     storage: Union[str, BaseStorage],
     host: str = "localhost",
     port: int = 8080,
+    artifact_store: Optional[ArtifactStore | ArtifactBackend] = None,
+    *,
     artifact_backend: Optional[ArtifactBackend] = None,
```

Comment:
This is a breaking change. I mean the following user code will occur errors.
```
run_server("xxx", "localhost", 8080, backend_obj)
```

Is this OK for optuna-dashboard? I don't have a strong opinion on this

Reply:
The third positional argument `artifact_store` accepts `ArtifactBackend` instances. So I think the suggested code won't be broken. What do you think?